Dockerfile.source

```
@@ -0,0 +1,21 @@
+# OCI image that packages the Skupper source tree for reproducible downstream builds.
+# Not intended to be run; publish to quay.io/skupper/skupper-source:<version> and
+# consume via build context / additional build contexts instead of git clone.
+ARG GO_IMAGE_BASE_TAG=1.25
+FROM golang:${GO_IMAGE_BASE_TAG} AS vendor
+
+WORKDIR /go/src/app
+COPY go.mod go.sum .
```

Hadolint (2.15.1)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)